apps/docs/app/ui/components/previewComponent/ComponentWrapper.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import dynamic from "next/dynamic";
+import { useMemo, useState } from "react";
+import clsx from "clsx";
+import { StyledSystemProvider } from "@hopper-ui/styled-system";
+import Card from "@/components/card/Card.tsx";
+import PreviewSkeleton from "@/app/ui/components/previewComponent/PreviewSkeleton.tsx";
+import ThemeSwitch from "@/components/themeSwitch/ThemeSwitch.tsx";
+import type { ColorScheme } from "@/context/theme/ThemeProvider.tsx";
+
+const ComponentWrapper = ({ key, className, name }: {
+    key: string;
+    className: string;
+    name: string;
+}) => {
+    const DynamicComponent = useMemo(() => dynamic(() => import(`../../../../../../packages/components/src/${name}.tsx`), {
```

Comment:
packages/components/src have subfolders under. I think this might cause a problem in the future.

Reply:
To avoid this problem, we manage it in the `content/components/COMPONENT.mdx` file by passing the path to the `PreviewComponent`. In this way, we can maintain flexibility in the folder hierarchy.

### Example
`<PreviewComponent name="buttons/docs/preview"/>`